src/components/PageElements/Elements/HistoryResponseElement.tsx

```diff
@@ -31,7 +27,22 @@ const HistoryResponseElement: React.FC<HistoryResponseElementProps> = ({
           />
           <p className="secondary date-indicator">{getSimplifiedTimeDifference(timeUpdated)}</p>
         </div>
-        <p>{message}</p>
+        {elementTypePluginCode ? (
+          <SummaryViewWrapper
+            element={
+              {
+                parameters,
+                pluginCode: elementTypePluginCode,
+                isRequired: true,
+                isVisible: true,
+              } as ElementState
+            }
+            response={response}
+            allResponses={{}}
```

Comment:
Wondering if this could be changed in `SummaryViewWrapper` so it's optional - if not passed we won't use to validate the element

Reply:
It's already not optional -- which is why I'm passing an empty object.

It's better this way, as SummaryViewWrapper can still do evaluations -- it's just that the "pre-evaluated parameters" now take priority.